_drafts/gallery.md

```diff
@@ -28,7 +28,7 @@ image:
 #
 # Metainformation & Customization
 #
-meta_description:
+description:
```

Comment:
I digged into the SEO-gem last week to. It adds JSON-LD into the site. which is a good think that isn't present in *Feeling Responsive*.